web/modules/custom/owner/config/install/core.entity_view_display.rdf_entity.owner.about.yml

```diff
@@ -0,0 +1,63 @@
+langcode: en
```

Comment:
It is fixed. But in all these cases I needed to manually edit code which was created by drush config-devel-export